patroni/ctl.py

```diff
@@ -646,3 +746,26 @@ def configure(config_file, dcs, namespace):
     config['dcs_api'] = str(dcs)
     config['namespace'] = str(namespace)
     store_config(config, config_file)
+
+
+@ctl.command('flush', help='Flush scheduled events')
+@click.argument('cluster_name')
+@click.argument('member_names', nargs=-1)
+@click.argument('target', type=click.Choice(['restart']))
+@click.option('--role', '-r', help='Flush only members with this role', default='any',
+              type=click.Choice(['master', 'replica', 'any']))
+@option_config_file
+@option_force
+@option_dcs
+def flush(cluster_name, member_names, config_file, dcs, force, role, target):
+    config, dcs, cluster = ctl_load_config(cluster_name, config_file, dcs)
+
+    members = get_members(cluster, cluster_name, member_names, role, force, 'flush')
+    for mn, member in members.items():
+        status = patroni_status(member, auth_header(config))
+        if target == 'restart':
+            if 'scheduled_restart' in status:
+                r = request_patroni('delete', member, 'restart', None, auth_header(config))
+                check_response(r, mn, 'flush scheduled restart')
```

Comment:
Should we also return some code to the user depending on whether the flush failed or not (since there is a force flag, it might could be used in a non-interactive script).